bot/exts/valentines/valentine_zodiac.py

```diff
@@ -13,6 +13,14 @@
 LETTER_EMOJI = ':love_letter:'
 HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
 
+zodiac_signs = ["Aries", "Taurus", "Gemini", "Cancer", "Leo", "Virgo", "Libra",
+                "Scorpio", "Sagittarius", "Capricorn", "Aquarius", "Pisces"]
+
+
+with open(Path("bot/resources/valentines/zodiac_explanation.json"), "r", encoding="utf8") as file:
+    """Load zodiac zodiac explanation from static JSON resource."""
+    zodiac_fact = load(file)
```

Comment:
there is already a file IO operation within the Class `ValentineZodiac` so it will be better to have both of them either inside the class as instance variables or outside the class as global variables.
I would say have it inside the class, and load the JSON with another method, say `load_zodiac_facts`
This is for consistency.